src/journal/journald-server.c

```diff
@@ -2928,3 +2973,30 @@ int config_parse_compress(
 
         return 0;
 }
+
+int config_parse_forward_to_socket(
+                const char* unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        SocketAddress* addr = data;
```

Comment:
`SocketAddress *addr = ASSERT_PTR(data);`

Reply:
It looks like config_parse_compress doesn't do these checks either so I'll add a commit for its assertions while I'm at it.